platform-db/src/main/java/com/softicar/platform/db/runtime/table/creator/DbAutomaticTableCreator.java

```diff
@@ -148,4 +156,15 @@ private IDbConnection getConnection() {
 		}
 		return connection;
 	}
+
+	private void createDatabaseIfMissing(IDbTable<?, ?> table) {
+
+		var databaseName = table.getFullName().getDatabaseName();
+		if (!existingDatabases.contains(databaseName.toLowerCase())) {
+			new DbStatement()//
+				.addText(String.format("CREATE SCHEMA IF NOT EXISTS `%s`", databaseName))
```

Comment:
I assume that the major performance benefit here comes from reducing the number of times this statement is executed. Is that correct?

Reply:
yes, that is the primary aim of this PR